ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java

```diff
@@ -1,364 +1,402 @@
-                                AdRequest.ERROR_CODE_INVALID_REQUEST);
-            }
-        }
-    }
-
-    private void createBanner() {
-        if (mVungleManager == null || !pendingRequestBanner.get())
-            return;
-
-        mVungleManager.cleanUpBanner(mPlacementForPlay);
-        vungleNativeAd = mVungleManager.getVungleNativeAd(mPlacementForPlay, mAdConfig, new VungleListener() {
-            @Override
-            void onAdEnd(String placement, boolean wasSuccessfulView, boolean wasCallToActionClicked) {
-                if (mMediationBannerListener != null) {
-                    if (wasCallToActionClicked) {
-                        // Only the call to action button is clickable for Vungle ads. So the
-                        // wasCallToActionClicked can be used for tracking clicks.
-                        mMediationBannerListener.onAdClicked(VungleInterstitialAdapter.this);
-                        mMediationBannerListener.onAdOpened(VungleInterstitialAdapter.this);
-                        mMediationBannerListener.onAdClosed(VungleInterstitialAdapter.this);
-                    }
-                }
-            }
-
-            @Override
-            void onAdStart(String placement) {
-                //let's load it again to mimic auto-cache, don't care about errors
-                mVungleManager.loadAd(placement, null);
-            }
-
-            @Override
-            void onAdFail(String placement) {
-                if (mMediationBannerListener != null) {
-                    mMediationBannerListener.onAdFailedToLoad(VungleInterstitialAdapter.this,
-                            AdRequest.ERROR_CODE_INTERNAL_ERROR);
-                }
-            }
-        });
-        View adView = vungleNativeAd != null ? vungleNativeAd.renderNativeView() : null;
-        if (adView != null) {
-            updateVisibility();
-            adLayout.addView(adView);
-            if (mMediationBannerListener != null) {
-                mMediationBannerListener.onAdLoaded(VungleInterstitialAdapter.this);
-            }
-        } else {
-            //missing resources
-            if (mMediationBannerListener != null) {
-                mMediationBannerListener.onAdFailedToLoad(VungleInterstitialAdapter.this,
-                        AdRequest.ERROR_CODE_INTERNAL_ERROR);
-            }
-        }
-    }
-
-    private void updateVisibility() {
-        if (vungleNativeAd != null) {
-            vungleNativeAd.setAdVisibility(!paused && visible);
-        }
-    }
-
-    @Override
-    public View getBannerView() {
-        Log.d(TAG, "getBannerView");
-        return adLayout;
-    }
-
-    private boolean hasBannerSizeAd(AdSize adSize) {
-        if (300 == adSize.getWidth() && 250 == adSize.getHeight()) {
-            mAdConfig.setAdSize(VUNGLE_MREC);
-            return true;
-        }
-        mAdConfig.setAdSize(VUNGLE_DEFAULT);
-        return false;
-    }
-
-}
+package com.vungle.mediation;

```

Comment:
unfortunately this is how the merge conflict appeared as entire two distinct code blocks